### PR TITLE
i18n(es): Translate `plugins`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translated `plugins` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/reference/plugins.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+El hook `setup` se puede utilizar para actualizar la configuración de Starlight o añadir integraciones de Astro.
+
+Este hook es llamado con las siguientes opciones:
+
+#### `config`
+
+**tipo:** `StarlightUserConfig`
+
+A read-only copy of the user-supplied [Starlight configuration](/reference/configuration).
+This configuration may have been updated by other plugins configured before the current one.
```
Comment (Villanuevand) [inline docs/src/content/docs/es/reference/plugins.md:58]: This part should be translated too.

**Review by Villanuevand — CHANGES_REQUESTED** [review]
There are some paragraphs you missed to translate. 
Everything else LGTM!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Villanuevand

File: docs/src/content/docs/es/reference/plugins.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+#### `updateConfig`
+
+**tipo:** `(newConfig: StarlightUserConfig) => void`
+
+Una función callback para actualizar la [configuración de Starlight](/es/reference/configuration).
+Proporciona las claves de configuración de nivel raíz que deseas sobreescribir.
+Para actualizar los valores de configuración anidados, debes proporcionar el objeto anidado completo.
+
+Para extender una opción de configuración existente sin sobreescribirla, extiende el valor existente en tu nuevo valor.
+En el siguiente ejemplo, una nueva cuenta de redes sociales se añade a la configuración existente de [`social`](/reference/configuration/#social) extendiendo `config.social` en el nuevo objeto `social`:
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/reference/plugins.md:69]:
```suggestion
En el siguiente ejemplo, se agrega una nueva cuenta en [`social`](/reference/configuration/#social) a la configuración existente extendiendo 'config.social' en el nuevo objeto social:
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Thanks @dreyfus92, I left a couple comments for clarity.

**Pull request merged**